docs/en/changelog.md

```diff
@@ -1,5 +1,64 @@
 ## Changelog
 
+### v2.23.0 (28/3/2022)
+
+#### Highlights
+
+- Support Mask2Former: [Masked-attention Mask Transformer for Universal Image Segmentation](https://arxiv.org/abs/2112.01527)
+- Support efficientnet: [EfficientNet: Rethinking Model Scaling for Convolutional Neural Networks](https://arxiv.org/abs/1905.11946)
+
+#### New Features
+
+- Support [Mask2Former](configs/mask2former)(#6938)(#7466)(#7471)
+- Support [EfficientNet](configs/efficientnet) (#7514)
+- Support setting data root through commands, so you don't have to modify the corresponding path in config files (#7386) 
+- Support setting different seeds to different ranks, so it can introduce more randomness into the training process (#7432)
+- Support training model with multiple machines, so you can train a model by launching with multiple machines simply connected with ethernet (#7415)
```

Comment:
Rephrase this line. Previously MMDet support training with multiple machines. We only update the dist_train.sh to so that the script can be used to support launching multi-node training on machines without slurm.